app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ArticlesController < ApplicationController
+  def index
+    @article = Article.all
+  end
+
+  def new
+    @article = Article.new
```

Comment:
I see that you use:
``` 
form_with model: Article.new ...
```
on your view. In this case, you don't need this variable. I would keep it though, and change my form to 
```
form_with model: @article
```